_pages/docs/assert-functions.md

```diff
@@ -7,25 +7,52 @@ permalink: /docs/assertions/
+* assert-at-least-one-equal($expected as item()\*, $actual as item()\*, $message as xs:string\*)
+* assert-same-values($expected as item()\*, $actual as item()\*)
+* assert-same-values($expected as item()\*, $actual as item()\*, $message as xs:string\*)
+* assert-equal($expected as item()\*, $actual as item()\*)
+* assert-equal($expected as item()\*, $actual as item()\*, $message as xs:string\*)
+* assert-not-equal($not-expected as item()\*, $actual as item()\*)
+* assert-not-equal($not-expected as item()\*, $actual as item()\*, $message as xs:string\*)
+* assert-equal-xml($expected, $actual)
+* assert-equal-xml($expected, $actual, $message as xs:string\*)
+* assert-equal-json($expected, $actual)
+* assert-equal-json($expected, $actual, $message as xs:string\*)
+* assert-equal-json-recursive($object1, $object2) as xs:boolean
+* assert-true($conditions as xs:boolean\*)
+* assert-true($conditions as xs:boolean\*, $message as xs:string?)
+* assert-false($conditions as xs:boolean\*)
+* assert-false($conditions as xs:boolean\*, $message as xs:string\*)
+* assert-meets-minimum-threshold($minimum as xs:decimal, $actual as xs:decimal+)
+* assert-meets-minimum-threshold($minimum as xs:decimal, $actual as xs:decimal+, $message as xs:string\*)
+* assert-meets-maximum-threshold($maximum as xs:decimal, $actual as xs:decimal+)
+* assert-meets-maximum-threshold($maximum as xs:decimal, $actual as xs:decimal+, $message as xs:string?)
+* assert-throws-error-with-message($function as xdmp:function, $expected-error-code as xs:string, $expected-message as xs:string)
+* assert-throws-error-with-message($function as xdmp:function, $expected-error-code as xs:string, $expected-message as xs:string, $message as xs:string\*)
 * assert-throws-error($function as xdmp:function)
 * assert-throws-error($function as xdmp:function, $error-code as xs:string?)
-* assert-throws-error($function as xdmp:function, $params as item()\*, $error-code as xs:string?)
-* assert-http-get-status($url as xs:string, $options as element(xdmp-http:options), $status-code)
+* assert-throws-error($function as xdmp:function, $param1 as item()\*, $error-code as xs:string?)
+* assert-throws-error($function as xdmp:function, $param1 as item()\*, $param2 as item()\*, $error-code as xs:string?)
+* assert-throws-error($function as xdmp:function, $param1 as item()\*, $param2 as item()\*, $param3 as item()\*, $error-code as xs:string?)
+* assert-throws-error($function as xdmp:function, $param1 as item()\*, $param2 as item()\*, $param3 as item()\*, $param4 as item()\*, $error-code as xs:string?)
+* assert-throws-error($function as xdmp:function, $param1 as item()\*, $param2 as item()\*, $param3 as item()\*, $param4 as item()\*, $param5 as item()\*, $error-code as xs:string?)
+* assert-throws-error($function as xdmp:function, $param1 as item()\*, $param2 as item()\*, $param3 as item()\*, $param4 as item()\*, $param5 as item()\*, $param6 as item()\*, $error-code as xs:string?)
```

Comment:
Let's not bother documenting that our assert-throws-error function only accepts up to 6 parameters.  Let's remove the six lines and revert back to the previous description:

> assert-throws-error($function as xdmp:function, $params as item()\*, $error-code as xs:string?)